domain/src/main/java/com/globant/equattrocchio/domain/models/ImageList.java

```diff
@@ -0,0 +1,19 @@
+package com.globant.equattrocchio.domain.models;
+
+import android.support.annotation.Nullable;
+
+import java.util.List;
+
+public class ImageList {
```

Comment:
This wrapping is useless, instead just use List<Image>

Reply:
Done